init-tools.cmd

```diff
@@ -24,6 +24,20 @@ if exist "%BUILD_TOOLS_SEMAPHORE%" (
   goto :EOF
 )
 
+if exist "%DotNetBuildToolsDir%" (
+  echo Using tools from '%DotNetBuildToolsDir%'.
+  mklink /j "%TOOLRUNTIME_DIR%" "%DotNetBuildToolsDir%"
+
+  if not exist "%DOTNET_CMD%" (
```

Comment:
Are we building into this that the path to the `dotnet.exe` needs to be underneath BuildTools?  Is that a valid assumption?

Reply:
We are building in that assumption but I'm also validating it by checking. I'm on the fence about making this assumption but without doing making it I don't know how to cleanly link things because the BuildTools is intertwined with the sdk under the Tools directory.

Reply:
Maybe adding the assumption to the error message?  "Ensure that `%DotNetBuildToolsDir%` contains the .NET Core SDK"?